tests/unit_tests/testlocalizer.cpp

```diff
@@ -295,7 +295,7 @@ void TestLocalizer::formattedDate_data() {
 
   QTest::addRow("en - one hour ago")
       << "en" << QDateTime(QDate(2000, 1, 1), QTime(10, 0), QTimeZone(0))
-      << QDateTime(QDate(2000, 1, 1), QTime(9, 0), QTimeZone(0)) << "9:00 AM"
```

Comment:
It strangely seems that 6.6 wants a _narrow_ no-break space here but changing this to a _narrow_ no-break white space (U+202F) causes this test to fail on Linux where we are still on 6.2.4. If anyone has immediate thoughts or guesses about why this is different in 6.6, I'd love to hear them.